### PR TITLE
br: update docs for gc life time

### PR DESCRIPTION
**Pull request by 3pointer**
### What is changed, added or deleted? (Required)
update br document about manually set the gc life time.
br can do it automatically after v4.0.3.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: br/backup-and-restore-tool.md [hunk 1]
```diff
@@ -122,16 +122,17 @@ BR 由多层命令组成。目前，BR 包含 `backup`、`restore` 和 `version`
...
-
-例如，将 `tikv_gc_life_time` 调整为 `720h`：
-
-{{< copyable "sql" >}}
-
-```sql
-mysql -h${TiDBIP} -P4000 -u${TIDB_USER} ${password_str} -Nse \
-    "update mysql.tidb set variable_value='720h' where variable_name='tikv_gc_life_time'";
-```
+> **注意：**
```
Comment (leoppro) [inline br/backup-and-restore-tool.md:125]: 我建议再加一句，自404起 br 已经支持 自适应gc，无需手动调整gc时间，等等。

不然没有背景知识的用户看这个主意会很懵

File: br/backup-and-restore-tool.md [hunk 2]
```diff
@@ -122,16 +122,19 @@ BR 由多层命令组成。目前，BR 包含 `backup`、`restore` 和 `version`
 
 使用 `br backup` 命令来备份集群数据。可选择添加 `full` 或 `table` 子命令来指定备份的范围：全部集群数据或单张表的数据。
 
-如果备份时间可能超过设定的 [`tikv_gc_life_time`](/garbage-collection-configuration.md#tikv_gc_life_time)（默认 `10m0s`，即表示 10 分钟），则需要将该参数调大。
-
-例如，将 `tikv_gc_life_time` 调整为 `720h`：
-
-{{< copyable "sql" >}}
+> **注意：**
+> 如果 br 版本小于 v4.0.3, 同时存在备份时间可能超过设定的 [`tikv_gc_life_time`](/garbage-collection-configuration.md#tikv_gc_life_time)（默认 `10m0s`，即表示 10 分钟）的情况下，需要手动将该参数调大。
```
Comment (TomShawn) [inline br/backup-and-restore-tool.md:126]:
```suggestion
> 如果 BR 的版本低于 v4.0.3，而且备份时间可能超过设定的 [`tikv_gc_life_time`](/garbage-collection-configuration.md#tikv_gc_life_time)（默认 `10m0s`，即表示 10 分钟），需要手动将该参数调大。
```

File: br/backup-and-restore-tool.md (hunk 1 above)
Comment (TomShawn) [reply br/backup-and-restore-tool.md:125]: PTAL again

File: br/backup-and-restore-tool.md [hunk 3]
```diff
@@ -122,16 +122,19 @@ BR 由多层命令组成。目前，BR 包含 `backup`、`restore` 和 `version`
...
+> ```sql
+> mysql -h${TiDBIP} -P4000 -u${TIDB_USER} ${password_str} -Nse \
+>     "update mysql.tidb set variable_value='720h' where variable_name='tikv_gc_life_time'";
+> ```
 
-```sql
-mysql -h${TiDBIP} -P4000 -u${TIDB_USER} ${password_str} -Nse \
-    "update mysql.tidb set variable_value='720h' where variable_name='tikv_gc_life_time'";
-```
+自 v4.0.3 起 br 已经支持自适应 gc，无需手动调整 tikv_gc_life_time.
```
Comment (TomShawn) [inline br/backup-and-restore-tool.md:137]:
```suggestion
自 v4.0.3 起 BR 已经支持自适应 GC，无需手动调整 `tikv_gc_life_time`.
```

**Review by leoppro — COMMENTED** [review]
LGTM

File: br/backup-and-restore-tool.md [hunk 4]
```diff
@@ -122,16 +122,20 @@ BR 由多层命令组成。目前，BR 包含 `backup`、`restore` 和 `version`
...
-如果备份时间可能超过设定的 [`tikv_gc_life_time`](/garbage-collection-configuration.md#tikv_gc_life_time)（默认 `10m0s`，即表示 10 分钟），则需要将该参数调大。
-
-例如，将 `tikv_gc_life_time` 调整为 `720h`：
+> **注意：**
+>
+> 如果 BR 的版本低于 v4.0.3，而且备份时间可能超过设定的 [`tikv_gc_life_time`](/garbage-collection-configuration.md#tikv_gc_life_time)（默认 `10m0s`，即表示 10 分钟），需要手动将该参数调大。
+>
+> 例如，将 `tikv_gc_life_time` 调整为 `720h`：
+>
+> {{< copyable "sql" >}}
```
Comment (TomShawn) [inline br/backup-and-restore-tool.md:132]:
```suggestion
>
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**